mimic/test/test_loadbalancer.py

```diff
@@ -111,25 +111,26 @@ def setUp(self):
         self.root = self.helper.root
         self.uri = self.helper.uri
 
-    def _create_loadbalancer(self, name=None, api_helper=None):
+    def _create_loadbalancer(self, name=None, api_helper=None, nodes=None):
```

Comment:
Added in https://github.com/rackerlabs/mimic/commit/f07ca2f6883802b19655ca360ba4b106972f869f